src/cosmosdb-preview/azext_cosmosdb_preview/commands.py

```diff
@@ -0,0 +1,81 @@
+    cf_db_accounts,
+    cf_restorable_database_accounts,
+    cf_restorable_sql_databases,
+    cf_restorable_sql_containers,
+    cf_restorable_sql_resources,
+    cf_restorable_mongodb_databases,
+    cf_restorable_mongodb_collections,
+    cf_restorable_mongodb_resources
+)
+
+
+def load_command_table(self, _):
+    cosmosdb_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#DatabaseAccountsOperations.{}',
+        client_factory=cf_db_accounts)
+
+    cosmosdb_restorable_database_accounts_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableDatabaseAccountsOperations.{}',
+        client_factory=cf_restorable_database_accounts)
+
+    cosmosdb_restorable_sql_databases_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableSqlDatabasesOperations.{}',
+        client_factory=cf_restorable_sql_databases)
+
+    cosmosdb_restorable_sql_containers_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableSqlContainersOperations.{}',
+        client_factory=cf_restorable_sql_containers)
+
+    cosmosdb_restorable_sql_resources_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableSqlResourcesOperations.{}',
+        client_factory=cf_restorable_sql_resources)
+
+    cosmosdb_restorable_mongodb_databases_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableMongodbDatabasesOperations.{}',
+        client_factory=cf_restorable_mongodb_databases)
+
+    cosmosdb_restorable_mongodb_collections_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableMongodbCollectionsOperations.{}',
+        client_factory=cf_restorable_mongodb_collections)
+
+    cosmosdb_restorable_mongodb_resources_sdk = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.operations#RestorableMongodbResourcesOperations.{}',
+        client_factory=cf_restorable_mongodb_resources)
+
+    with self.command_group('cosmosdb restorable-database-account', cosmosdb_restorable_database_accounts_sdk, client_factory=cf_restorable_database_accounts, is_preview=True) as g:
+        g.show_command('show', 'get_by_location')
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb', cosmosdb_sdk, client_factory=cf_db_accounts) as g:
+        g.show_command('show', 'get')
+        g.custom_command('restore', 'cli_cosmosdb_restore', is_preview=True)
+        g.custom_command('create', 'cli_cosmosdb_create')
+        g.custom_command('update', 'cli_cosmosdb_update')
+        g.custom_command('list', 'cli_cosmosdb_list')
```

Comment:
you could only add `restore` if only this one is new command. And cli would combine the command `restore` in extension and other commands in azure cli main repo.

But if you want to change the function used for existing commands, you could keep it.

Reply:
Restore is the new command I've added.
But I've also modified the behavior of show, create, update and list commands.  (Added new properties).

So I have all these commands here, so that all of them will be overridden.